packages/styleguide/src/components/ColorLabel/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from 'glamor'
+import React, { ReactNode } from 'react'
+
+// TODO: these should be out of the SG and usable by story components
+export const COLOR_LABEL_EVENTS = {
+  enter: 'enterColorLabel',
+  leave: 'leaveColorLabel',
+}
+
+export const ColorLabel = ({
+  color,
+  children,
+}: {
+  color: string
+  children: ReactNode
+}) => {
+  const enterEvent = new CustomEvent(COLOR_LABEL_EVENTS.enter, {
+    bubbles: true,
+    detail: { color },
+  })
+  const leaveEvent = new CustomEvent(COLOR_LABEL_EVENTS.leave, {
+    bubbles: true,
+    detail: { color },
```

Comment:
Note for later: don't just pass color value here (use an ID instead). Color should come from a color palette that also has dark color scheme support.

Reply:
agreed. 🙂 
this refactoring only makes sense once the color stuff is out of the styleguide, though.